NEWS.md

```diff
@@ -4,6 +4,40 @@ PgBouncer changelog
 PgBouncer 1.22.x
 ----------------
 
+**2024-06-28  -  PgBouncer 1.22.2  -  "Into the new beginnings"**
+- Features
+  * Add support for user name maps for for `cert` and `peer` authentication
+    methods. ([#996])
+  * Add support for negotiating the postgres protocol version with the client. ([#1007])
+  * Add support for replication connections through PgBouncer. ([#876])
+  * Add support for rolling restarts when peering is configured. ([#902])
```

Comment:
This change included a breaking change, which should be listed at the top of the changelog. I suggest to move the whole:

SIGTERM doesn't cause immediate shutdown of the PgBouncer process anymore. It now does a "super safe shutdown": waiting for all clients to disconnect before shutting down. If you used SIGTERM to trigger an immediate shutdown in your Dockerfile or SystemD service file, you should replace that signal with a SIGQUIT to keep the existing behaviour. The new SIGTERM behaviour allows rolling restarts of multiple PgBouncer processes behind a load balancer, or listening on the same port using `so_reuseport`. ([#902])

Reply:
Should we have a BREAKING CHANGES section?

Reply:
Yeah, I think that would be good. No need for ALL CAPS though.